resources/views/forms/components/video-embed-preview.blade.php

```diff
@@ -1,10 +1,17 @@
 @php
     $state = $getState();
+
     $params = [
         'autoplay' => $state['autoplay'] ? 1 : 0,
+        'mute' => $state['mute'] ? 1 : 0,
         'loop' => $state['loop'] ? 1 : 0,
         'controls' => $state['controls'] ? 1 : 0,
     ];
+
+    if ($state['embed_url'] && Str::contains($state['embed_url'], 'youtube')) {
+        $videoId = Str::after($state['embed_url'], 'https://www.youtube.com/embed/');
```

Comment:
Heb je geprobeerd of je met `$getId()` van op `lijn 18` niet aan het id kan? of is dat een ander id?

Reply:
Had ik nog niet geprobeerd, maar als dit eens dump krijg ik iets anders terug. In had dit nu met de hero's getest en dan krijg ik 'data.hero.video' terug terwijl ik eigenlijk het id van youtube / vimeo video wil

Reply:
Zal dat inderdaad niet de video id zijn.